README.md

```diff
@@ -54,31 +94,43 @@ This module provisions a dataset and a table with an associated JSON schema.
 <!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
 
 ## Requirements
-### Terraform plugins
-- [Terraform](https://www.terraform.io/downloads.html) 0.12.x
-- [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v2.5.0
-
-### Permissions
-In order to execute this module you must have a Service Account with the following roles:
- - roles/bigquery.dataOwner
-
-#### Script Helper
-A helper script for configuring a Service Account is located at (./helpers/setup-sa.sh).
-
-## Install
-### Terraform
-Be sure you have the current Terraform version (0.12.x), you can choose the binary from [Terraform releases](https://releases.hashicorp.com/terraform/).
-
-### kitchen-terraform
-To set this up on your machine, follow the official [Kitchen installation](https://github.com/newcontext-oss/kitchen-terraform) instructions.
-- Kitchen tests are located: [test/integration/full](test/integration/full).
-- Terraform fixtures are located: [test/fixtures/full](test/fixtures/full).
-
-## Running tests
-`cd /path/to/terraform-google-bigquery`
-The following command will run all tests for the module:
-`make`
-
-### macOS mojave notes
-To run kitchen tests on macOS > 10.14.4 xcode will need to be [reset](https://apple.stackexchange.com/questions/254380/why-am-i-getting-an-invalid-active-developer-path-when-attempting-to-use-git-a)
-`xcode-select --install`
+
+These sections describe requirements for using this module.
+
+### Software
+
+The following dependencies must be available:
+
+- [Terraform][terraform] v0.12
+- [Terraform Provider for GCP][terraform-provider-gcp] plugin v2.0
```

Comment:
We should provide the minor version number (v2.5).

Reply:
Updated